x-pack/platform/plugins/shared/alerting/server/application/backfill/methods/find/find_backfill.ts

```diff
@@ -77,6 +77,9 @@ export async function findBackfill(
     if (params.end) {
       timeFilters.push(`ad_hoc_run_params.attributes.end <= "${params.end}"`);
     }
+    if (params.initiator) {
```

Comment:
the query schema allows initiatorID to be passed, should we also add a filter for it if it's defined?

Reply:
Added!